plugin/evm/vm_validators_test.go

```diff
@@ -76,84 +78,50 @@ func TestValidatorState(t *testing.T) {
 	)
 	require.NoError(err, "error initializing GenesisVM")
 
-	// Test case 1: state should not be populated until bootstrapped
```

Comment:
this test isn't really super relevant anymore as `state` is no longer externally exposed -- this changes a lot of what can be tested (and a lot of tested in AvalancheGo now), so this test is kind of awkward now. thoughts?